entoas/policy/policy.go

```diff
@@ -0,0 +1,9 @@
+package policy
+
+const (
+	None Policy = iota
+	Exclude
+	Expose
+)
+
+type Policy uint
```

Comment:
Creating a package for a few lines of code does not feel right. Why not hold this on `entoas`:

```go
entoas.PolicyNone, entoas.PolicyExclude, entoas.PolicyExpose
```